### DISCUSSION TITLE
Cannot ssh to MC cluster login node

### DISCUSSION THREAD

**Discussion by mflynn-lanl** [microbiomedata/nmdc-edge#276] [General]
@edwins I created an MC cluster using snapshot:
`9a74382b-86e0-422a-bd05-21895b8e8b67`
The cluster was successfully created but I was not able to ssh to the instance.

To troubleshoot the problem I used the console from Horizon for instance:
https://js2.jetstream-cloud.org/project/instances/4f9ece16-d38d-4391-a958-d8b00e0506af/
Here's a screenshot
![Screenshot 2024-08-12 at 3 15 44 PM](https://github.com/user-attachments/assets/8f910769-ac38-4ce6-9219-78ba5c6cf0f8)

**Comment by edwins** [comment]
That is unusual. If the deployment went to active, then the kernel panic likely happened after successfully the completing the deployment process. I'm not sure why that might happen. 

Were both vms AlmaLinux9?

What is the deployment id of the mc deployment?

**Reply by mflynn-lanl** [reply]
I can't get the deployment id because I get a 500 error when I go to the cacao page

**Reply by mflynn-lanl** [reply]
Yes, they are all AlmaLinux9. Any idea on what's wrong with Cacao? I filed a ticket with Jetstream2 yesterday evening but I haven't heard anything

**Reply by edwins** [reply]
I have to get access-ci to change my email address...

Can you clear your cookies for cacao.jetstream-cloud.org?

**Reply by mflynn-lanl** [reply]
I cleared my cookies and that fixed the problem. I'm not sure what the deployment id is for MC:
```
os-lat-clust IU

Started: 8/12/2024, 5:00:46 PM MDT, Build time: 5 min, Updated: 8/12/2024, 5:05:55 PM MDT
Template: Magic Castle, Digital Research Alliance  Template ID: template-cl26m8j761e2g21s4sig
Credential: cacao_MCB180107_1708459319_
```

**Reply by edwins** [reply]
I underlined the deployment id in red -- I'll put it to my ui designer to see if there's something we can do to make that clearer in the future:

![image](https://github.com/user-attachments/assets/1ca8a4e7-6b7e-4db5-83b9-ed3b479fbcfa)

**Reply by mflynn-lanl** [reply]
`deployment-cqt987hogabsfg6esl10`

**Comment by mflynn-lanl** [comment]
I've been doing some trial-and-error to isolate the problem:
trial 1:
AlmaLinux9, no volume snapshot
successful cluster creation, successful ssh, cannot su user

trial 2:
AlmaLinux9, volume snapshot
refdata-snapshot 03b12b41-4d4b-499f-b797-6be289a23b97
unsuccessful cluster creation

trial 3:
AlmaLinux9
refdata-save-4 2173a856-507f-49bd-9858-a9806784125b
successful cluster creation, successful ssh, successful su user

Seems like there's a problem with the second volume snapshot I created using an AlmaLinux9 VM. Also, I'm not able to ssh to the clusters I created using the earlier images I created which had the software I installed

**Reply by mflynn-lanl** [reply]
I created a new image snapshot and now I can successfully create clusters using my new volume snapshot
